### PR TITLE
docs(irsa): switch keycloak to automated docker way

### PR DESCRIPTION
**Pull request by Langleu**
## Description

Switches the suggested `Dockerfile` of the AWS IRSA documentation for a more automated way by utilizing Gradle.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by maxdanilov — COMMENTED** [review]
LGTM, just making a suggestion to Gradle file improvement.

File: docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md [hunk 1]
```diff
@@ -129,18 +129,67 @@ Required are the following `software.amazon.awssdk` artifacts for the `aws-advan
...
+Create a file called `build.gradle` with the following content:
+
+```java
+apply plugin: 'groovy'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```
[the remark this answers is not in the document]
Comment (Langleu) [reply docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md:145]: nice suggestion!
probably wouldn't go with a full renovate suite here.
Longterm would be nice to have a separate repository with some tests implemented to make sure that everything is still working as expected.

[the remark this answers is not in the document]
Comment (Langleu) [reply docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md:148]: included the change 👍

**Review by maxdanilov — APPROVED** [review]
Looks good, thank you 🙇‍♂️

**Pull request merged**